src/Umbraco.Web.UI.Client/src/common/directives/components/content/umbcontentnodeinfo.directive.js

```diff
@@ -32,11 +32,15 @@
                 // get document type details
                 scope.documentType = getDocumentType(scope.node);
 
-                loadAuditTrail();
-
+                // load audit trail on tab change
+                eventsService.on("tab change", function (event, args) {
+                    if (args.context.innerHTML === "Properties" && args.context.hash === "#tab0") {
```

Comment:
I wonder if this will work for other backoffice localizations, e.g. Danish :)

![image](https://user-images.githubusercontent.com/2919859/30696750-0158b658-9edd-11e7-81a3-7e25ccb043de.png)

Reply:
Good question :), will look into that 👍 .